Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/LargeSubmitAsync/LargeSubmitAsyncClient.cs

```diff
@@ -208,19 +208,30 @@ public override void EntryPoint()
                                                  int                 workloadTimeInMs,
                                                  CancellationToken   token = default)
   {
-    var result = Enumerable.Range(0,
-                                  nbTasks)
-                           .Batch(nbTasks / Props.MaxParallelChannels)
-                           .AsParallel()
-                           .Select(bucket => bucket.Select(_ => service.SubmitAsync("ComputeSum",
-                                                                                    ParamsHelper(numbers,
-                                                                                                 workloadTimeInMs),
-                                                                                    this,
-                                                                                    token)));
-
-    var taskIds = result.SelectMany(t => Task.WhenAll(t)
-                                             .Result);
-    return taskIds.ToList();
+    var resultQueries = Enumerable.Range(0,
+                                         nbTasks)
+                                  .Batch(nbTasks / Props.MaxParallelChannels)
+                                  .AsParallel();
```

Comment:
Maybe this would make a good first use for ParallelSelect.

Reply:
Perhaps later but not for this PR.